test/101_cross_hosts_2_test.sh

```diff
@@ -0,0 +1,50 @@
+#! /bin/bash
+
+. "$(dirname "$0")/config.sh"
+
+C1=10.2.1.4
+C2=10.2.1.7
+UNIVERSE=10.2.0.0/16
+SUBNET_1=10.2.2.0/24
```

Comment:
I'd be in favour of removing these to reduce the amount of "noise" then, as one may think they are required for this test case.